swarmrl/observables/concentration_field.py

```diff
@@ -135,3 +135,37 @@ def compute_observable(self, colloids: List[Colloid]):
         ]
 
         return np.array(observables).reshape(-1, 1)
+
+
+class ConcentrationFieldAbsoluteValue(ConcentrationField, Observable, ABC):
+    """
+    Position in box observable. For the absolute value no
+    initialization and no historic_positions are needed,
+    but still doing it this way makes the code more swapable
+    from relative to absolute values.
+
+    Attributes
+    ----------
+    historic_positions : dict
+            A dictionary of past positions of the colloid to be used in the gradient
+            computation.
+    """
+
+    def compute_single_observable(self, index: int, colloids: List[Colloid]) -> float:
+        """
+        Compute the observable for a single colloid.
+
+        Parameters
+        ----------
+        index : int
+                Index of the colloid to compute the observable for.
+        colloids : List[Colloid]
+                List of colloids in the system.
+        """
+        reference_colloid = colloids[index]
+        position = onp.copy(reference_colloid.pos) / self.box_length
```

Comment:
the box length scaling here is weird. It should be the users responsibility to scale the decay-function if they want this behaviour.

Reply:
This is my fault code-wise but I do agree, we can update these things in all observables / tasks.

Reply:
Is somebody already working on the update for all the observables?